examples/apollo/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "react-cosmos-apollo-example",
+  "version": "2.0.0-beta.28",
+  "description": "Example of mocking the Apollo Client",
+  "repository": "https://github.com/react-cosmos/react-cosmos/tree/master/examples/apollo",
+  "license": "MIT",
+  "scripts": {
+    "start": "NODE_ENV=development cosmos"
+  },
+  "dependencies": {
+    "react-apollo": "^1.4.15",
+    "react-cosmos-utils": "^2.0.0-beta.28",
```

Comment:
Sorry, mate! This should also be `2.0.0-beta.24`. Only packages that have changes get bumped on publish, so while latest is `2.0.0-beta.28`, lower level packages sometimes stay behind.

Reply:
Haha, no probs!